amxmodx/srvcmd.cpp

```diff
@@ -63,6 +63,44 @@ void amx_command()
 
 		print_srvconsole("%d plugins, %d running\n", plugins, running);
 	}
+	else if (!strcmp(cmd, "plugin"))
+	{
+		if (CMD_ARGC() < 3)
+		{
+			print_srvconsole("Usage: amxx plugin [ id ]\nFor a list of plugins, use the \"amxx plugins\" command\n");
+		}
+		else
+		{
+			auto id = atoi(CMD_ARGV(2));
+			auto plugin = g_plugins.findPlugin(id);
+
+			if (plugin && plugin->isValid())
+			{
+				print_srvconsole("   Name: %s\n", plugin->getTitle());
+				print_srvconsole("   Version: %s\n", plugin->getVersion());
+				print_srvconsole("   Author: %s\n", plugin->getAuthor());
+
+				auto url = plugin->getDescription();
```

Comment:
Oh, copy-paste. How many programms you ruined, how many bugs introduced.. How many hours of debugging I spent unable to stop you..

Please read this line carefully.

Reply:
lol :D